pytest/lib/key.py

```diff
@@ -38,10 +38,12 @@ def from_json_file(cls, filename: str):
             return cls.from_json(json.load(rd))
 
     @classmethod
-    def from_seed_testonly(cls, account_id: str, seed: str) -> 'Key':
+    def from_seed_testonly(cls, account_id: str, seed: str = None) -> 'Key':
         """
         Deterministically produce an **insecure** signer pair from a seed.
```

Comment:
We can also mention in the docstring that if the seed is not specified, it will be derived from the account id.

Reply:
yeah that makes sense, added a sentence just now